docs/en_US/TrialExample/Trials.md

```diff
@@ -127,6 +127,14 @@ In the YAML configure file, you need to set *useAnnotation* to true to enable NN
 useAnnotation: true
 ```
 
+## Standalone mode for debug
+
+NNI supports standalone mode for trial code to run without starting an NNI experiment. This is for finding out bugs in trial code more conveniently. NNI annotation natively supports standalone mode, as the added NNI related lines are comments. For NNI trial APIs, in standalone mode, some APIs return fake/empty values, for example, `nni.get_next_parameter` will return `{}` (NOTE: for users please assign default values to the hyperparameters in your trial code). And some other APIs, such as `nni.report_final_result`, do not really report values. 
```

Comment:
some API? I think we should make it clear.

Reply:
thanks, will make it clear